server/server.go

```diff
@@ -18,9 +18,9 @@ limitations under the License.
 // Insights results aggregator service. In current version, the following
 // REST API endpoints are available:
 //
-// API_PREFIX/organization - list of all organizations (HTTP GET)
+// API_PREFIX/organizations - list of all organizations (HTTP GET)
 //
-// API_PREFIX/cluster/{organization} - list of all clusters for given organizations (HTTP GET)
```

Comment:
> list of all clusters for given organizations (HTTP GET)
Does it mean that I can provide more than one orgID separated by a comma and receive a list of clusters?

Reply:
nope, should be organizations/organization